kubectl-cloudflow/docker/client.go

```diff
@@ -136,3 +156,50 @@ func imageMatchesNameAndTag(image types.ImageSummary, imageNameAndTag string) bo
 	}
 	return false
 }
+
+// GetStreamletDescriptorsForImage extracts the configuration of a Cloudflow label from a docker image
+// and the image with digest in a struct
+func GetStreamletDescriptorsForImage(cli *client.Client, imageName string) (domain.CloudflowStreamletDescriptorsDigestPair, string) {
+	images, err := cli.ImageList(context.Background(), types.ImageListOptions{})
+
+	streamletDescriptorsLabelName := "com.lightbend.cloudflow.streamlet-descriptors"
+
+	if err != nil {
+		util.LogAndExit("Failed to list local docker images, %s", err.Error())
```

Comment:
I think it makes more sense for private methods not to exit, but return an error.

Reply:
We should only LogAndExit in handling the commands on top level.

Reply:
fixed for this instance .. there may be others (not part of this PR) which we can clean up globally in a separate PR